edi_storage/components/base.py

```diff
@@ -36,17 +36,19 @@ def _dir_by_state(self, direction, state):
         """
         assert direction in ("input", "output")
         assert state in ("pending", "done", "error")
+        if absolute:
+            return PurePath(self.backend[direction + "_dir_" + state] or "")
         return PurePath((self.backend[direction + "_dir_" + state] or "").strip(" /"))
```

Comment:
@simahawk I did not know exactly why you were stripping the path here, I use absolute paths in the edi backend configuration (input_dir_pending, input_dir_done...). Any clarification or recommendation?

Reply:
Done. no need to keep absolute now, thanks!